lib/std/special_paths.nim

```diff
@@ -0,0 +1,29 @@
+#[
```

Comment:
moved it out of lib/std and put it here: `testament/lib/stdtest/specialpaths.nim` along with `tests/nim.cfg` that sets `--path:"../testament/lib"` so that:
* testament tests can simply `import stdtest/foo` without messing with error prone relative paths ( `"../../foo/bar"`)
* there is 0 confusion possible with stdlib modolues

moving it to `tests/stdlib/specialpaths.nim` and adding `tests/stdlib` to path would not be good as it'd bring in package scope things that aren't meant to be imported. 
Other general purpose helper modules that don't belong in stdlib but are useful for testament tests can go also under `testament/lib/stdtest/`